backend/src/modules/match-module/core/domain/event/MatchHasStarted.ts

```diff
@@ -0,0 +1,16 @@
+import { MatchId } from '../MatchId';
+import { MatchSideId } from '../MatchSideId';
+
+export class MatchHasStarted {
+  readonly occurredAt: Date;
+  readonly matchId: string;
+  readonly firstTeamId: string;
```

Comment:
To może słownictwo fiestMatchSide będzie lepsze :)?

Reply:
w takim razie zmienić w całym module, żeby była jakaś spójność? :)

Reply:
No zdaje mi się, że w obrębie tego modłu możemy używać matchSide. Nie mamy świadomoście teamów. Bo stronie jest albo gracz albo team, albo cokolwiek co da się jednosnacznie zidentyfikwoac :)

Reply:
w porządku w takim razie wszędzie będzie firstMatchSideId i secondMatchSideId :P